content/tracing/setup/_index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Setup
+kind: Documentation
+further_reading:
+- link: "/tracing/setup/docker"
+  tag: "Documentation"
+  text: Docker setup
+- link: "/tracing/setup/go"
+  tag: "Documentation"
+  text: Go language instrumentation
+- link: "/tracing/setup/java"
+  tag: "Documentation"
+  text: Java language instrumentation
+- link: "/tracing/setup/python"
+  tag: "Documentation"
+  text: Python language instrumentation
+- link: "/tracing/setup/ruby"
+  tag: "Documentation"
+  text: Ruby language instrumentation
+---
+
+## Setup process
+
+With our infrastructure monitoring, metrics are sent to the Agent, which then forwards them to Datadog. Similarly, tracing metrics are also sent to the Agent: The application code instrumentation flushes to the Agent every 1 s ([see here for the Python client](https://github.com/DataDog/dd-trace-py/blob/69693dc7cdaed3a2b6a855325109fa100e42e254/ddtrace/writer.py#L159) for instance) and the Agent flushes to the [Datadog API every 10s](https://github.com/DataDog/datadog-trace-agent/blob/master/config/agent.go#L170).  
+
+To start tracing your application:
+
+1. **Install the Datadog Agent**:    
+  Install and configure the latest [Datadog Agent](https://app.datadoghq.com/account/settings#agent) (version 6.0 or above is required). For additional information, reference the [getting started guide](https://github.com/DataDog/datadog-trace-agent/tree/master/config#agent-configuration).
+
+2. **Install the Trace Agent**:  
+  Depending on your operating system, install the Datadog Trace Agent separately:
+  * On **Linux**, and **[Docker](/tracing/setup/docker)** the Trace Agent is pre-packaged with the standard Datadog Agent and no extra configuration is needed.
+
+  * On **macOS** and **Windows**, install and run the [Trace Agent](https://github.com/DataDog/datadog-trace-agent) in addition to the Datadog Agent.  
```

Comment:
For Windows it is part of the packaged Agent too.

Reply:
@LotharSee so it's just macOS that needs to have the trace-agent?